test/typecheck/testdata/good/_ignore/BUILD

```diff
@@ -0,0 +1,14 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
+
+go_library(
```

Comment:
agh, this shouldn't exist, maybe we need stub BUILD files above this with the gazelle comment?